target/linux/brcm47xx/image/Makefile

```diff
@@ -519,7 +519,16 @@ define Device/usrobotics-usr5461
 endef
 TARGET_DEVICES += usrobotics-usr5461
 
-TARGET_DEVICES += standard standard-noloader-gz
+define Device/legacy-standard
+  $(Device/standard)
+endef
+TARGET_DEVICES += legacy-standard
+
+define Device/legacy-standard-noloader-gz
+  $(Device/standard-noloader-gz)
+endef
+TARGET_DEVICES += legacy-standard-noloader-gz
+
```

Comment:
What's this, these devices are renamed? This looks like an unrelated change that should be a separate commit.

Reply:
It isn't really unrelated, it is just really hard to get from the diff why it is done.

At the moment we have the same image referenced twice. One time in `ifeq ($(SUBTARGET),legacy)` and one time in `ifeq ($(SUBTARGET),mips74k)`.

With this change it is more obvious that the same images should be build for both subtargets.

But I agree, that this change should go into an extra commit due to the rename of the images. In fact, I dropped this chunk from my local branch which I compile test at the moment.

Reply:
Exactly, the name of the generated images remains the same, it's just a cosmetic change to identify better the subtarget, and to have a  device defined for every append to TARGET_DEVICES.
I understand that the change should be on a separate commit. Do I add another commit or should I create another pull request?

Without defining the devices, the change would be:
```diff
-TARGET_DEVICES += standard standard-noloader-gz
+TARGET_DEVICES += standard
+TARGET_DEVICES += standard-noloader-gz
```